.babelrc

```diff
@@ -1 +1,4 @@
-{ "presets": ["es2015"] }
+{
+  "presets": ["es2015"],
+  "plugins": ["add-module-exports"]
```

Comment:
Without the plugin, all `export default xxx` module need to be import with `require('xxx').default`